<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Map">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>22</y>
      <width>751</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Up">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вверх</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Влево</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Down">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вниз</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Right">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вправо</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
